cves/kernel/CVE-2012-2745.yml

```diff
@@ -151,10 +142,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: was found on 2012-04-09, looked at multiple changelogs,repositoriesa and cve.mitre links but no information on how it was found
+  automated: false
+  contest: false
+  developer: false
```

Comment:
Could you include how you know the person who found the bug was not a developer?

Reply:
Included new information to clarify.